tests/integration/features/servicemesh_feature_test.go

```diff
@@ -189,11 +266,63 @@ func createSMCPInCluster(cfg *rest.Config, smcpObj *unstructured.Unstructured, n
 		return err
 	}
 
-	r, err := dynamicClient.Resource(gvr.SMCP).Namespace(namespace).UpdateStatus(context.TODO(), result, metav1.UpdateOptions{})
+	_, err = dynamicClient.Resource(gvr.SMCP).Namespace(namespace).UpdateStatus(context.TODO(), result, metav1.UpdateOptions{})
 	if err != nil {
 		return err
 	}
-	fmt.Printf("result: %v", r)
 
 	return nil
 }
+
+func getServiceMeshControlPlane(cfg *rest.Config, namespace, name string) (*unstructured.Unstructured, error) {
+	dynamicClient, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		return nil, err
+	}
+
+	smcp, err := dynamicClient.Resource(gvr.SMCP).Namespace(namespace).Get(context.TODO(), name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return smcp, nil
+}
+
+func fromTestTmpDir(fileName string) string {
```

Comment:
This is temporary and will be removed when https://github.com/opendatahub-io/opendatahub-operator/pull/797 gets merged